backend/src/notifications/notifications.service.ts

```
@@ -20,15 +27,28 @@ export class NotificationsService {
   async sendNotifications(
     userId: number,
     letters: Letter[],
-    recipients: string[],
+    createNotificationDto: CreateNotificationDto,
   ): Promise<Notification[]> {
     const templatedMessages = await this.getTemplatedMessages(userId, letters)
-    const sentMessages = await this.smsNotificationService.send(
-      templatedMessages,
-      recipients,
-    )
-
-    return this.notificationRepository.save(sentMessages)
+    switch (createNotificationDto.notificationType) {
+      case CitizenNotificationMethod.SMS: {
+        const sentMessages = await this.smsNotificationService.send(
+          templatedMessages,
+          createNotificationDto.recipients,
+        )
+        return this.notificationRepository.save(sentMessages)
+      }
+      case CitizenNotificationMethod.EMAIL: {
+        const sentEmails =
+          await this.emailNotificationService.sendEmailNotification(
+            templatedMessages,
+            createNotificationDto.recipients,
+          )
+        return this.notificationRepository.save(sentEmails)
+      }
+      default:
+        return []
```

just for consideration, should we be throwing an error here instead? not relevant for now since the code is relatively lean and simple, but might make for easier debugging in the future instead of failing silently.

---

Hmm makes sense, makes sense!